enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/OtpService.java

```diff
@@ -165,8 +176,9 @@ public OtpVerifyResponse verifyOtpCode(String processId, String otpCode, OtpType
         OtpVerifyResponse response = new OtpVerifyResponse();
         response.setProcessId(processId);
         response.setOnboardingStatus(process.getStatus());
+        response.setExpired(expired);
         response.setVerified(verified);
-        response.setRemainingAttempts(remainingAttempts);
+        response.setRemainingAttempts(expired ? 0 : maxFailedAttempts - failedAttempts);
```

Comment:
I am not sure if this is correct - we are losing information "expired with 3 remaining attempts"

Reply:
Fixed